org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -66,6 +66,18 @@ public enum TargetProperty {
                 config.auth = ASTUtils.toBoolean(value);
             }),
 
+    /**
+     * Path of configuration file of SST. It allows including OpenSSL libraries 
+     * and use SST for authentication and authorization.
```

Comment:
```suggestion
     * and using SST for authentication and authorization.
```

Reply:
Thanks for your comments. This PR is not finished yet, and I made it to check the changed files. Thanks for the changes. I'll request a review when I'm finished.😁